File: wipe.js
process.env.DATABASE_URL = "file:./dev.db";
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient({
    datasourceUrl: process.env.DATABASE_URL
});

async function main() {
    await prisma.notification.deleteMany();
    await prisma.queueBooking.deleteMany();
    await prisma.user.deleteMany({
        where: { role: 'member' }
    });
    console.log("Wiped all members, bookings, and notifications.");
}

main()
    .catch(e => {
        console.error(e);
        process.exit(1);
    })
    .finally(async () => {
        await prisma.$disconnect();
    });

File: wipe.py
import sqlite3

conn = sqlite3.connect('dev.db')
cursor = conn.cursor()

# Wipe notifications
cursor.execute('DELETE FROM Notification')
print("Deleted Notifications:", cursor.rowcount)

# Wipe queue bookings
cursor.execute('DELETE FROM QueueBooking')
print("Deleted QueueBookings:", cursor.rowcount)

# Wipe users with role member
cursor.execute("DELETE FROM User WHERE role='member'")
print("Deleted Users (members):", cursor.rowcount)

conn.commit()
conn.close()
print("Wiped all members, bookings, and notifications.")
